include/MeshKernel/CurvilinearGrid.hpp

```diff
@@ -54,6 +54,24 @@ namespace meshkernel
             Invalid        //(0)
         };
 
+        /// @brief A struct describing the column and row indices of a node
+        struct NodeIndices
+        {
+            /// @brief Overloads equality with another NodeIndices
+            bool operator==(const NodeIndices& rhs) const
```

Comment:
if you use c++20 you can let the compiler make this operator
https://en.cppreference.com/w/cpp/language/default_comparisons

Reply:
Using c++20 could be considered, needs to be investigated if it can be used in our TeamCity agents (namely an update of the boost libraries might be required)